### PR TITLE
Registry changes

### PR DESCRIPTION
**Pull request by raamb**
- Read tags from service metadata instead of registry contract
- Reme references to types repository from registry contract

### CODE REVIEW

**Review by anandrgitnirman — APPROVED** [review]

File: packages/sdk/snet/sdk/metadata_provider/ipfs_metadata_provider.py [hunk 1]
```diff
@@ -12,8 +12,7 @@ def __init__(self, ipfs_client, registry_contract):
         self._ipfs_client = ipfs_client
 
     def fetch_org_metadata(self, org_id):
-        (found, id, metadata_uri, owner, members, service_ids,
-         repository_ids) = self.registry_contract.functions.getOrganizationById(
+        (found, id, metadata_uri, owner, members, service_ids) = self.registry_contract.functions.getOrganizationById(
             bytes(org_id, "utf-8")).call()
         if found is not True:
             raise Exception('No  organization is foubd "{}"'.format(org_id))
```
Comment (prashantramangupta) [inline packages/sdk/snet/sdk/metadata_provider/ipfs_metadata_provider.py:18]: We can correct this Exception message.

File: packages/snet_cli/snet/snet_cli/metadata/service.py [hunk 2]
```diff
@@ -167,6 +169,26 @@ def remove_group(self, group_name):
...
 
+    def get_tags(self):
+        tags = []
+        if "tags" in self.m:
+            tags = self.m["tags"]
+        return tags
+
+    def add_tag(self, tag_name):
+        if (tag_name in self.m["tags"]):
+            print("the tag \"%s\" is already present" %
```
Comment (prashantramangupta) [inline packages/snet_cli/snet/snet_cli/metadata/service.py:180]: String formatting structure is different. It should be the same across all code base.
print(f"the tag {tag_name} is already present").

File: packages/sdk/snet/sdk/metadata_provider/ipfs_metadata_provider.py (hunk 1 above)
Comment (raamb) [reply packages/sdk/snet/sdk/metadata_provider/ipfs_metadata_provider.py]: Done

File: packages/snet_cli/snet/snet_cli/metadata/service.py (hunk 2 above)
Comment (raamb) [reply packages/snet_cli/snet/snet_cli/metadata/service.py]: Done

File: packages/snet_cli/snet_cli/commands/mpe_service.py [hunk 3]
```diff
@@ -276,21 +275,27 @@ def publish_metadata_in_ipfs_and_update_registration(self):
...
+    def metadata_remove_tags(self):
+        metadata = load_mpe_service_metadata(self.args.metadata_file)
+        [metadata.remove_tag(tag) for tag in self.args.tags]
+        metadata.save_pretty(self.args.metadata_file)
 
     def update_registration_add_tags(self):
-        params = self._get_params_for_tags_update()
-        self.transact_contract_command(
-            "Registry", "addTagsToServiceRegistration", params)
+        print("This command has been deprecated. Please use `snet service metadata-add-tags` instead")
```
[the remark this answers is not in the document]
Comment (raamb) [reply packages/snet_cli/snet_cli/commands/mpe_service.py]: Done

**Review by prashantramangupta — APPROVED** [review]
Approved

**Pull request merged**